src/cli/commands/deploy/deploy.ts

```diff
@@ -161,6 +166,7 @@ export async function deploy(options: SWACLIConfig) {
     appLocation,
     outputLocation: resolvedOutputLocation,
     apiLocation: resolvedApiLocation,
+    dataApiLocation
```

Comment:
Any change do we need to do in the readWorkflowFile for enabling it to handle dataApiLocation ?

Reply:
Thanks Sulabh, I've adjusted this as well. I noticed that this feature currently broken while dev'ing and testing. I will let the bugfix take place in another PR.

Reply:
yup make sense. @sgollapudi77 please create a tracking task if we don't have it already.

Reply:
Added a task now Sulabh.